app/views/teaching_events/about_git_events.html.erb

```diff
@@ -47,17 +47,47 @@
         image_alt: "A woman at a Get Into Teaching event wearing a blue T-shirt with Here to Help written on it.",
       ) %>
 
-      <section class="category__cards">
-        <%= tag.nav(class: "category__nav-cards") do %>
-          <ul class="one-column horizontal">
-            <%= render(Categories::CardComponent.with_collection(categorise_events(@git_events))) %>
-          </ul>
-        <% end %>
-        <div>
-          <h2 class="heading-s">Can't find what you're looking for?</h2>
-          <p>You can also take a look at our <%= link_to("online Q&A events", events_path({ teaching_events_search: { type: ["onlineqa"] }}), class: "link--black") %> or <%= link_to("teacher training provider events", events_path({ teaching_events_search: { type: ["provider"] }}), class: "link--black") %> in your area.</p>
-        </div>
-      </section>
+      <% if @git_events.any? %>
+        <section class="category__cards">
+          <%= tag.nav(class: "category__nav-cards") do %>
+            <ul class="one-column horizontal">
+              <%= render(Categories::CardComponent.with_collection(categorise_events(@git_events))) %>
+            </ul>
+          <% end %>
+          <div>
+            <h2 class="heading-s">Can't find what you're looking for?</h2>
+            <p>You can also take a look at our <%= link_to("online Q&A events", events_path({ teaching_events_search: { type: ["onlineqa"] }}), class: "link--black") %> or <%= link_to("teacher training provider events", events_path({ teaching_events_search: { type: ["provider"] }}), class: "link--black") %> in your area.</p>
+          </div>
+        </section>
+      <% else %>
+        <section class="col col-space-m">
+          <div class="col-640">
+            <p><strong>Our autumn events are now closed. Details of our spring events will be published on this page in early January 2023.</strong></p>
```

Comment:
```suggestion
            <p><strong>Our autumn Get Into Teaching events are now closed. Details of our spring events will be published on this page in early January 2023.</strong></p>
```

@philspray I think it might be worth specifying `Get Into Teaching` events here as otherwise the following sentence doesn't make sense (our events are closed so look at our other events)?